--- a/vendor/fakerphp/faker/src/Faker/Provider/Image.php	2020-12-18 17:50:48.000000000 +0100
+++ b/vendor/fakerphp/faker/src/Faker/Provider/ImageModif.php	2022-10-18 16:35:25.656622199 +0200
@@ -8,7 +8,7 @@
 class Image extends Base
 {
     /** @var string */
-    public const BASE_URL = 'https://via.placeholder.com';
+    public const BASE_URL = 'https://api.lorem.space/image';
 
     /**
      * @var array
@@ -43,27 +43,11 @@
         $word = null,
         $gray = false
     ) {
-        $size = sprintf('%dx%d.png', $width, $height);
-
-        $imageParts = [];
-        if ($category !== null) {
-            $imageParts[] = $category;
-        }
-        if ($word !== null) {
-            $imageParts[] = $word;
-        }
-        if ($randomize === true) {
-            $imageParts[] = Lorem::word();
-        }
-
-        $backgroundColor = $gray === true ? 'CCCCCC' : str_replace('#', '', Color::safeHexColor());
-
         return sprintf(
-            '%s/%s/%s%s',
+            '%s?w=%s&h=%s',
             self::BASE_URL,
-            $size,
-            $backgroundColor,
-            count($imageParts) > 0 ? '?text=' . urlencode(implode(' ', $imageParts)) : ''
+            $width,
+            $height
         );
     }
 
@@ -104,6 +88,7 @@
             $fp = fopen($filepath, 'w');
             $ch = curl_init($url);
             curl_setopt($ch, CURLOPT_FILE, $fp);
+            curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
             $success = curl_exec($ch) && curl_getinfo($ch, CURLINFO_HTTP_CODE) === 200;
             fclose($fp);
             curl_close($ch);
